docs/scanners/gitleaks.md

```diff
@@ -0,0 +1,43 @@
+# [Gitleaks](https://github.com/zricethezav/gitleaks)
+
+The [Gitleaks Scanner](https://github.com/zricethezav/gitleaks) tool for finding secrets within a directory or Git repo.
+
+## Configuration
+
+Gitleaks currently always runs with the `--recursive` and `--format json` options turned on.
```

Comment:
Did you mean "always runs" or "by default runs"?
If you specify `--format sarif`, then GitLeaks will produce a sarif only without the json, correct?